packages/frint-cli/package.json

```diff
@@ -2,15 +2,14 @@
   "name": "frint-cli",
   "version": "5.2.0",
   "description": "CLI for Frint",
-  "main": "lib/index.js",
   "homepage": "https://github.com/frintjs/frint/tree/master/packages/frint-cli",
   "bin": {
-    "frint": "./bin/frint.js"
+    "frint": "./lib/bin/frint.js"
   },
   "scripts": {
-    "lint": "cross-env ../../node_modules/.bin/eslint --color '{root,commands,bin}/**/*.js'",
-    "transpile": "echo \"Nothing to transpile\"",
-    "test": "cross-env ../../node_modules/.bin/mocha --colors --recursive ./commands/*.spec.js",
+    "lint": "cross-env ./node_modules/.bin/tslint -p .",
+    "transpile": "cross-env ./node_modules/.bin/tsc",
+    "test": "cross-env ../../node_modules/.bin/mocha --require ts-node/register --colors --recursive ./src/**/*.spec.ts",
```

Comment:
There are multiple ways to run TS unit tests.  
We could also simply `transpile` first and then run `mocha` on the transpiled JS, but this option to use `ts-node` seems to be more popular (and this way `watch` can work properly too).